.github/workflows/docker-indexer-grpc.yaml

```diff
@@ -0,0 +1,61 @@
+name: "Docker Indexer gRPC"
```

Comment:
rename to "Docker Indexer gRPC test" to avoid confusion this for building indexer docker